spec/lib/declare_schema/generator_spec.rb

```diff
@@ -19,13 +19,23 @@ class Alpha::Beta < #{active_record_base_class}
       end
     EOS
 
-    expect_model_definition_to_eq('alpha', <<~EOS)
-      module Alpha
-        def self.table_name_prefix
-          'alpha_'
+    if ActiveSupport::VERSION::MAJOR >= 7
+      expect_model_definition_to_eq('alpha', <<~EOS)
+        module Alpha
+          def self.table_name_prefix
+            "alpha_"
```

Comment:
😞 It's unfortunate that we need to separate out blocks of tests like this to account for quotation usage differences. Maybe in the future we can find a better way to test that doesn't care which quotes are used.